### PR TITLE
Add pytorch prepackaged server document

### PR DESCRIPTION
**Pull request by jackklpan**
Signed-off-by: Jack Pan <jackpan@infuseai.io>

### CODE REVIEW

File: docs/model-deployment-prepackaged-server-pytorch.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    self.model.load_state_dict(torch.load(model_file))
+    self.model.eval()
+```
+
+And, when a client send the request, it will call the predict function:
+```python
+def predict(self, X):
+    with torch.no_grad():
+        inputs = torch.from_numpy(X).float()
+        return self.model(inputs).tolist()
```
Comment (popcornylu) [inline docs/model-deployment-prepackaged-server-pytorch.md:57]: remove float() and change toList() to numpy()

**Review by popcornylu — COMMENTED** [review]
Please add the following information

Add Model Infomration
1. Latest image:
1. Input: Numpy
1. Output: Numpy
1. Model Files: structures, https://pytorch.org/tutorials/beginner/saving_loading_models.html

File: docs/model-deployment-prepackaged-server-pytorch.md (hunk 1 above)
Comment (jackklpan) [reply docs/model-deployment-prepackaged-server-pytorch.md]: Fixed. Thx!

**Comment by popcornylu** [comment]
LGTM

**Pull request merged**